magpie-persist/src/main/java/io/openraven/magpie/plugins/persist/AssetModel.java

```diff
@@ -0,0 +1,162 @@
+package io.openraven.magpie.plugins.persist;
+
+import java.time.Instant;
+
+public class AssetModel {
```

Comment:
Why not just use MagpieResource?

Reply:
I also was thinking about that, coz thats even more simpler .  But I have two concerns:
1. Usage of MagpieResource is bounding with its builder which requires ObjectMapper. 
So in order to create a simple DTO from content I will need to create Object mapper which is not required actually to just simply persist the data.
2. AssetModel in that case associate class naming with table name, so its directly pointing to table structure and table source and doesn't implies any other usages except getting data out of table or saving from that object to table.

If we will use different DTOs on different layers we will achieve less coupling, and will be able to design them respectively to layer requirements (as for example MagpieResource has toJson method providing serialization to ObjectNode, which is not required on persistence layer either).

But if we dont need this, and MagpieResource entity should be treated as asset row I could drop AssetModel. Coz if we will add fields in MagpieResources they will not automatically means that table has such new attributes (tradeoff)

@kickroot @stefansjfw @PrzemyslawTusinski  what do you think?

Reply:
Let's keep it decoupled.

Reply:
I'd still go with using MagpieResource. 
Every time we will add new field to MagpieResource we will update flyway and upsert method to handle this asset

Reply:
I get that  it is more code, but I'm strongly adverse to having the db schema coupled to the resource definition.  I think the way @belosh59 has done it is the best approach.